web/test/integration/project.feature

```diff
@@ -0,0 +1,11 @@
+Feature:  Save the project json in db 
```

Comment:
This does not describe the epic/result we are expecting, I would I suggest something like `Display projects on the homepage`

Reply:
okay will change the text